text/machine_comprehension/gpt-2/README.md

```diff
@@ -0,0 +1,73 @@
+# GPT-2
+
+## Use-cases
+Transformer-based language model for text generation.
+
+## Description
+[GPT-2](https://openai.com/blog/better-language-models/) is a large transformer-based language model with a simple objective: predict the next word, given all of the previous words within some text.
+
+## Model
+
+ |Model        |Download  |Checksum| Download (with sample test data)|ONNX version|Opset version|Accuracy |
+|-------------|:--------------|:--------------|:--------------|:--------------|:--------------|:--------------|
+|GPT-2       |[522.81 MB](https://onnxzoo.blob.core.windows.net/models/opset_10/GPT2/model.onnx) | [MD5](https://onnxzoo.blob.core.windows.net/models/opset_10/GPT2/gpt2-md5.txt)| [438.3 MB](https://onnxzoo.blob.core.windows.net/models/opset_10/GPT2/GPT-2.tar.gz)| 1.6 | 10 |mAP of [0.024](https://docs.google.com/spreadsheets/d/1sryqufw2D0XlUH4sq3e9Wnxu5EAQkaohzrJbd5HdQ_w/edit#gid=0)|

```

Comment:
links should be to the github file

Reply:
I was following the same pattern we had for previous models like Faster RCNN https://github.com/onnx/models/blob/8d50e3f598e6d5c67c7c7253e5a203a26e731a1b/vision/object_detection_segmentation/faster-rcnn/README.md
But I can change that. Do you mean both model and md5 files?

Reply:
for fasterrcnn, the model was not put in Git LFS because it wasnt enabled back then. take a look at BERT for a more recent example after LFS was enabled,